server/public/utils/sql/sql_utils.go

```diff
@@ -64,13 +64,15 @@ func SetupConnection(logger mlog.LoggerIFace, connType string, dataSource string
 		mlog.String("dataSource", sanitized),
 	)
 
-	for i := 0; i < attempts; i++ {
-		logger.Info("Pinging SQL")
+	for attempt := 1; attempt <= attempts; attempt++ {
+		if attempt > 1 {
+			logger.Info("Pinging SQL", mlog.Int("attempt", attempt))
```

Comment:
This was the last remaining "superfluous" log during startup. Figured squelching the first shouldn't be an issue, even in production.